### PR TITLE
OSDOCS-8376: Migrate "Getting Started with Rosa - Deploy the Cluster (HCP)"

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-8376](https://issues.redhat.com//browse/OSDOCS-8376): Migrate "Getting Started with Rosa - Deploy the Cluster (HCP)" from rosaworkshop.io into product documentation.

Version(s):
4.14+
Issue:
https://issues.redhat.com/browse/OSDOCS-8376

Link to docs preview:
https://68291--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-hcp

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @yuwang-RH! Can I please get a QE review of this PR? Thank you!

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-hcp.adoc [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+=== Creating a VPC
+. First, ensure that your AWS CLI (`aws`) is configured to use a region where ROSA with HCP is available. To find out which regions are supported run the following command:
++
+[source,terminal]
+----
+rosa list regions --hosted-cp
+----
+    
+. Create the VPC. For this workshop, the following script will create the VPC and its required components for you. It will use the region configured for the `aws` CLI.
```
Comment (dupliaka) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-hcp.adoc:38]: workshop

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-hcp.adoc [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
++
+[source,terminal]
+----
+Public Subnet: subnet-0faeeeb0000000000
+Private Subnet: subnet-011fe340000000000
+----
+
+=== Creating your OIDC configuration
+
+In this workshop, we will use the automatic mode when creating the OIDC configuration. We will also store the OIDC ID as an environment variable for later use. The command uses the ROSA CLI to create your cluster's unique OIDC configuration.
```
Comment (dupliaka) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-hcp.adoc:72]: Workshop leftover

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-hcp.adoc [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+=== Creating your OIDC configuration
+
+In this workshop, we will use the automatic mode when creating the OIDC configuration. We will also store the OIDC ID as an environment variable for later use. The command uses the ROSA CLI to create your cluster's unique OIDC configuration. 
+
+* To create the OIDC configuration for this workshop, run the following command:
++
+[source,terminal]
+----
+export OIDC_ID=$(rosa create oidc-config --mode auto --managed --yes -o json | jq -r '.id')
```
Comment (dupliaka) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-hcp.adoc:78]: Workshop leftover

**Review by dupliaka — CHANGES_REQUESTED** [review]
Thanks for the change, however as a QE I need more clarification on this new section, please follow https://issues.redhat.com/browse/OSDOCS-8376

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-hcp.adoc [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+----
+rosa create cluster --cluster-name $CLUSTER_NAME \
+    --subnet-ids ${PUBLIC_SUBNET_ID},${PRIVATE_SUBNET_ID} \
+    --hosted-cp \
+    --region $REGION \
+    --oidc-config-id $OIDC_ID \
+    --sts --mode auto --yes
+----
+
+The control plane and API are ready after about 10 minutes and the worker nodes and cluster are ready and completely useable after about 15 to 20 minutes. The cluster will have a control plane across three AWS availability zones in your selected region and create two worker nodes in your AWS account.
```
Comment (dupliaka) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-hcp.adoc:113]: That conflicts with Provisioning time mentioned in:
 https://68291--docspreview.netlify.app/openshift-rosa/latest/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly#rosa-hcp-classic-comparison_rosa-hcp-sts-creating-a-cluster-quickly

**Comment by jneczypor** [comment]
Hi @dupliaka - I made the suggested edits to the tutorial. Is there anything else that needs to be changed in this HCP tutorial before I move it to peer review? Thanks!

**Review by dupliaka — APPROVED** [review]
Thanks @jneczypor, approved for Hosted Control Planes guide

**Pull request merged**